docs/api/Apify.md

```diff
@@ -82,7 +82,7 @@ effect.
 <td colspan="3"><p>Idempotency key enables you to ensure that a webhook will not be added multiple times in case of
   an actor restart or other situation that would cause the <code>addWebhook()</code> function to be called again.
   We suggest using the actor run ID as the idempotency key. You can get the run ID by calling
-  [`Apify.getEnv()](apify#module_Apify.getEnv) function.</p>
+  <a href="apify#module_Apify.getEnv">`Apify.getEnv()</a> function.</p>
```

Comment:
What was wrong with the previous link?

Reply:
Aside from the fact that it was missing an ending ` in the name of the link

Reply:
I am not aware of any changes to the docs. I have only run the `build-docs` cmd, so not sure.